.travis.yml

```diff
@@ -10,17 +13,12 @@ node_js:
     - "7.8.0"
 
 addons:
-  firefox: "45.0"
+  firefox: "66.0"
```

Comment:
is the firefox addon needed if we use chrome webdriver?

Reply:
You're right, it's not needed, an in fact Firefox install slows down init stage of a Travis instance. Removed it.